src/lunch_bot/handler.clj

```diff
@@ -24,7 +31,7 @@
     (cond
       (= com "random") (yelp/get-random)
       (= com "vote") (vote/start user text)
-      :else "What do you want?")))
+      :else (handle-query-request (body "text")))))
```

Comment:
(= com "search") (handle-query-request (body "text")))
      :else "What do you want?")